File: Day-16/Pattern25.py
n = int(input())

def PatternTwentyFive(n):
    i = 1
    printNum = 1

    while(i <=n):
        space = i -1
        while(space):
            print(' ', end='')
            space -=1
        
        num = n-i +1
        while(num):
            print(printNum, end='')
            num-=1
            printNum += 1

        i+=1
        print(end='\n')
    
PatternTwentyFive(n)

File: Day-17/Pattern25.js
function PatternTwentyFive(n) {
  let i = 1;
  let printNum = 1;

  while (i <= n) {
    let str = "";

    let spaces = i - 1;
    while (spaces) {
      str += " ";
      spaces--;
    }

    let nums = n - i + 1;
    while (nums) {
      str += printNum;
      printNum++;
      nums--;
    }
    i++;
    console.log(str);
  }
}

PatternTwentyFive(4);
